지미_김지명/2주차_미션/mission2/src/App.tsx

```diff
@@ -0,0 +1,13 @@
+import './App.css'
+import ContextPage from './useContext/ContextPage'
+
+function App() {
+
+  return (
+    <>
+    <ContextPage></ContextPage>
```

Comment:
ContextPage 하나만 지금 렌더링 하니 굳이 <></>로 감쌀 필요 없이 그냥 

```suggestion
  return (
    <ContextPage />
  )
```

라고만 작성해주셔도 됩니다!